app/models/post.rb

```diff
@@ -0,0 +1,16 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  has_many :comments, dependent: :destroy
+  has_many :likes, dependent: :destroy
+
+  validates :title, presence: true
+  validates :text, presence: true
+
+  def recent_comments(limit = 5)
+    comments.order(created_at: :desc).limit(limit)
+  end
+
+  def update_user_posts_counter
+    user.update(posts_counter: user.posts.count)
```

Comment:
- Good job done so far, but since the User table is referenced in the posts table as `author` or `author_id`, using `user` here would return an error, I suggest you use `author.update(posts_counter: author.posts.count)`. Please fix this 🙏